terraform/modules/task-definitions/signon/main.tf

```diff
@@ -15,6 +15,10 @@ provider "aws" {
   }
 }
 
+locals {
+  app_name = "signon"
+}
+
```

Comment:
I'd merge this with the existing locals block further down (similarly for all the other occurrences). Could even consider reusing local.service_name as well, if it fits in most cases - no big deal either way on that one though because we'll need to take a holistic look at these various app-name-related variables at some point soon (but not now).